doofinder/includes/api/class-management-api.php

```diff
@@ -132,7 +133,7 @@ private function sendRequest( $url, $data ) {
 	public function updateItem( $item_id, $name, $data ) {
 		$this->log->log( 'Update item' );
 
-		return $this->sendRequest( "$this->api_host/api/v2/search_engines/$this->hash/indices/$name/items/$item_id", array(
+		return $this->sendRequest( $this->api_host.API_VERSION_SE."/$this->hash/indices/$name/items/$item_id", array(
```

Comment:
@eduardogomez97 are not we going to have the same problem with `$this->hash`???

I think in PHP you can use `{}` for object properties…

```php
"{$this->api_host}/api/v2/search_engines/{$this->hash}/indices/$name/items/$item_id"
```

Also, instead of a class, I would create an `apiURL` method:

```php
// $this->apiURL("/search_engines/{$this->hash}/indices/$name/items/$item_id");
public function apiURL($path) {
  return "{$this->api_host}/api/v2$path";
}
```

Reply:
Done!